common/BUILD

```diff
@@ -17,22 +17,16 @@
 #
 
 package(default_visibility = ["//visibility:public"])
-load("@graknlabs_rules_deployment//maven:rules.bzl", "deploy_maven_jar")
+load("//dependencies/maven:rules.bzl", "deploy_maven_jar")
 
 java_library(
     name = "common",
-    srcs = [":grakn-version"] + glob(
-        ["**/*.java"],
-        exclude = [
-            "**/test/*.java",
-            "templates/**",
-        ]
-    ),
-    deps = [
-        "//dependencies/maven/artifacts/com/google/auto/value:auto-value",
-        "//dependencies/maven/artifacts/com/google/code/findbugs:annotations",
-        "//dependencies/maven/artifacts/com/google/guava:guava",
-        "//dependencies/maven/artifacts/org/slf4j:slf4j-api",
+    srcs = ["//common/templates/util:grakn-version"],
```

Comment:
This source `//common/templates/util:grakn-version` should go into sources of `//common/util`

Reply:
@haikalpribadi fixed, see https://github.com/vmax/grakn/blob/split-common-to-multiple-pkgs/common/util/BUILD#L4